### ISSUE TITLE
Wrong "oldest backup timestamp" in info output

### ISSUE THREAD

**Issue by terrorobe** [pgmasters/backrest#120]
The oldest backup timestamp should refer to the timestamp of the full backup which should be around Aug 3rd, 13:36

```
backrest@prd-ba-01-tor ~ $ /opt/backrest/bin/pg_backrest info --stanza prd-wiki-02-tor
stanza prd-wiki-02-tor
    status: ok
    oldest backup label: 20150803-143249F
    oldest backup timestamp: 2015-08-04 13:36:12
    latest backup label: 20150803-143249F_20150804-030007D
    latest backup timestamp: 2015-08-04 13:36:12
backrest@prd-ba-01-tor ~ $ ls -l /data/postgresql/backups/backup/prd-wiki-02-tor/
total 4
drwxr-xr-x 3 backrest backrest   51 Aug  3 14:32 20150803-143249F
drwxrwxr-x 3 backrest backrest   51 Aug  4 03:00 20150803-143249F_20150804-030007D
-rw-r--r-- 1 backrest backrest 2690 Aug  4 03:00 backup.info
lrwxrwxrwx 1 backrest backrest   33 Aug  4 03:00 latest -> 20150803-143249F_20150804-030007D
backrest@prd-ba-01-tor ~ $ cat /data/postgresql/backups/backup/prd-wiki-02-tor/backup.info
[backrest]
backrest-checksum="1ee96ebcf2cb5a7d8e49fa777867cb89abf2181f"
backrest-format=4
backrest-version="0.78"

[backup:current]
20150803-143249F={"backrest-format":4,"backrest-version":"0.78","backup-archive-start":"00000001000001D5000000F9","backup-archive-stop":"00000001000001D5000000F9","backup-info-repo-size":52067565,"backup-info-repo-size-delta":52067565,"backup-info-size":146629818,"backup-info-size-delta":146629818,"backup-timestamp-start":1438612360,"backup-timestamp-stop":1438612369,"backup-type":"full","db-id":1,"option-archive-check":true,"option-archive-copy":true,"option-compress":true,"option-hardlink":false,"option-start-stop":true}
20150803-143249F_20150804-030007D={"backrest-format":4,"backrest-version":"0.78","backup-archive-start":"00000001000001D600000090","backup-archive-stop":"00000001000001D600000090","backup-info-repo-size":53225222,"backup-info-repo-size-delta":12299461,"backup-info-size":146646202,"backup-info-size-delta":59116744,"backup-prior":"20150803-143249F","backup-reference":["20150803-143249F"],"backup-timestamp-start":1438657202,"backup-timestamp-stop":1438657207,"backup-type":"diff","db-id":1,"option-archive-check":true,"option-archive-copy":true,"option-compress":true,"option-hardlink":false,"option-start-stop":true}

[backup:history]
20150803-143249F={"backrest-format":4,"backrest-version":"0.78","backup-archive-start":"00000001000001D5000000F9","backup-archive-stop":"00000001000001D5000000F9","backup-info-repo-size":52067565,"backup-info-repo-size-delta":52067565,"backup-info-size":146629818,"backup-info-size-delta":146629818,"backup-timestamp-start":1438612360,"backup-timestamp-stop":1438612369,"backup-type":"full","db-id":1,"option-archive-check":true,"option-archive-copy":true,"option-compress":true,"option-hardlink":false,"option-start-stop":true}
20150803-143249F_20150804-030007D={"backrest-format":4,"backrest-version":"0.78","backup-archive-start":"00000001000001D600000090","backup-archive-stop":"00000001000001D600000090","backup-info-repo-size":53225222,"backup-info-repo-size-delta":12299461,"backup-info-size":146646202,"backup-info-size-delta":59116744,"backup-prior":"20150803-143249F","backup-reference":["20150803-143249F"],"backup-timestamp-start":1438657202,"backup-timestamp-stop":1438657207,"backup-type":"diff","db-id":1,"option-archive-check":true,"option-archive-copy":true,"option-compress":true,"option-hardlink":false,"option-start-stop":true}

[db]
db-catalog-version=201306121
db-control-version=937
db-id=1
db-system-id=6023966503359999454
db-version="9.3"

[db:history]
1={"db-catalog-version":201306121,"db-control-version":937,"db-system-id":6023966503359999454,"db-version":"9.3"}
backrest@prd-ba-01-tor ~ $
```

**Comment by dwsteele** [comment]
This one is not jumping out at me.  Can you run:

/opt/backrest/bin/pg_backrest info --stanza prd-wiki-02-tor --output json

I want to see if that structure is getting loaded correctly.  Thanks!

**Comment by terrorobe** [comment]
Sorry for the late reply - got lost in the mailbox.

Here's info, info with json and manifest dump:

```
backrest@prd-ba-01-tor ~ $ /opt/backrest/bin/pg_backrest info --stanza prd-wiki-02-tor
stanza prd-wiki-02-tor
    status: ok
    oldest backup label: 20150803-143249F
    oldest backup timestamp: 2015-08-05 17:01:36
    latest backup label: 20150803-143249F_20150805-030007D
    latest backup timestamp: 2015-08-05 17:01:36
backrest@prd-ba-01-tor ~ $ /opt/backrest/bin/pg_backrest info --stanza prd-wiki-02-tor --output json
[
    {
        "backup" : [
            {
                "archive" : {
                    "start" : "00000001000001D5000000F9",
                    "stop" : "00000001000001D5000000F9"
                },
                "backrest" : {
                    "format" : 4,
                    "version" : "0.78"
                },
                "database" : {
                    "id" : 1
                },
                "info" : {
                    "delta" : 146629818,
                    "repository" : {
                        "delta" : 52067565,
                        "size" : 52067565
                    },
                    "size" : 146629818
                },
                "label" : "20150803-143249F",
                "prior" : null,
                "reference" : null,
                "timestamp" : {
                    "start" : 1438612360,
                    "stop" : 1438612369
                },
                "type" : "full"
            },
            {
                "archive" : {
                    "start" : "00000001000001D600000090",
                    "stop" : "00000001000001D600000090"
                },
                "backrest" : {
                    "format" : 4,
                    "version" : "0.78"
                },
                "database" : {
                    "id" : 1
                },
                "info" : {
                    "delta" : 59116744,
                    "repository" : {
                        "delta" : 12299461,
                        "size" : 53225222
                    },
                    "size" : 146646202
                },
                "label" : "20150803-143249F_20150804-030007D",
                "prior" : "20150803-143249F",
                "reference" : [
                    "20150803-143249F"
                ],
                "timestamp" : {
                    "start" : 1438657202,
                    "stop" : 1438657207
                },
                "type" : "diff"
            },
            {
                "archive" : {
                    "start" : "00000001000001D7000000B1",
                    "stop" : "00000001000001D7000000B1"
                },
                "backrest" : {
                    "format" : 4,
                    "version" : "0.78"
                },
                "database" : {
                    "id" : 1
                },
                "info" : {
                    "delta" : 60255432,
                    "repository" : {
                        "delta" : 11379901,
                        "size" : 52190861
                    },
                    "size" : 146687162
                },
                "label" : "20150803-143249F_20150805-030007D",
                "prior" : "20150803-143249F",
                "reference" : [
                    "20150803-143249F"
                ],
                "timestamp" : {
                    "start" : 1438743602,
                    "stop" : 1438743607
                },
                "type" : "diff"
            }
        ],
        "db" : [
            {
                "id" : "1",
                "system-id" : 6023966503359999454,
                "version" : "9.3"
            }
        ],
        "name" : "prd-wiki-02-tor",
        "status" : {
            "code" : 0,
            "message" : "ok"
        }
    }
]
backrest@prd-ba-01-tor ~ $ cat /data/postgresql/backups/backup/prd-wiki-02-tor/backup.info
[backrest]
backrest-checksum="4dc998e206c1a19743ce8cd19ff677f27a1f75dd"
backrest-format=4
backrest-version="0.78"

[backup:current]
20150803-143249F={"backrest-format":4,"backrest-version":"0.78","backup-archive-start":"00000001000001D5000000F9","backup-archive-stop":"00000001000001D5000000F9","backup-info-repo-size":52067565,"backup-info-repo-size-delta":52067565,"backup-info-size":146629818,"backup-info-size-delta":146629818,"backup-timestamp-start":1438612360,"backup-timestamp-stop":1438612369,"backup-type":"full","db-id":1,"option-archive-check":true,"option-archive-copy":true,"option-compress":true,"option-hardlink":false,"option-start-stop":true}
20150803-143249F_20150804-030007D={"backrest-format":4,"backrest-version":"0.78","backup-archive-start":"00000001000001D600000090","backup-archive-stop":"00000001000001D600000090","backup-info-repo-size":53225222,"backup-info-repo-size-delta":12299461,"backup-info-size":146646202,"backup-info-size-delta":59116744,"backup-prior":"20150803-143249F","backup-reference":["20150803-143249F"],"backup-timestamp-start":1438657202,"backup-timestamp-stop":1438657207,"backup-type":"diff","db-id":1,"option-archive-check":true,"option-archive-copy":true,"option-compress":true,"option-hardlink":false,"option-start-stop":true}
20150803-143249F_20150805-030007D={"backrest-format":4,"backrest-version":"0.78","backup-archive-start":"00000001000001D7000000B1","backup-archive-stop":"00000001000001D7000000B1","backup-info-repo-size":52190861,"backup-info-repo-size-delta":11379901,"backup-info-size":146687162,"backup-info-size-delta":60255432,"backup-prior":"20150803-143249F","backup-reference":["20150803-143249F"],"backup-timestamp-start":1438743602,"backup-timestamp-stop":1438743607,"backup-type":"diff","db-id":1,"option-archive-check":true,"option-archive-copy":true,"option-compress":true,"option-hardlink":false,"option-start-stop":true}

[backup:history]
20150803-143249F={"backrest-format":4,"backrest-version":"0.78","backup-archive-start":"00000001000001D5000000F9","backup-archive-stop":"00000001000001D5000000F9","backup-info-repo-size":52067565,"backup-info-repo-size-delta":52067565,"backup-info-size":146629818,"backup-info-size-delta":146629818,"backup-timestamp-start":1438612360,"backup-timestamp-stop":1438612369,"backup-type":"full","db-id":1,"option-archive-check":true,"option-archive-copy":true,"option-compress":true,"option-hardlink":false,"option-start-stop":true}
20150803-143249F_20150804-030007D={"backrest-format":4,"backrest-version":"0.78","backup-archive-start":"00000001000001D600000090","backup-archive-stop":"00000001000001D600000090","backup-info-repo-size":53225222,"backup-info-repo-size-delta":12299461,"backup-info-size":146646202,"backup-info-size-delta":59116744,"backup-prior":"20150803-143249F","backup-reference":["20150803-143249F"],"backup-timestamp-start":1438657202,"backup-timestamp-stop":1438657207,"backup-type":"diff","db-id":1,"option-archive-check":true,"option-archive-copy":true,"option-compress":true,"option-hardlink":false,"option-start-stop":true}
20150803-143249F_20150805-030007D={"backrest-format":4,"backrest-version":"0.78","backup-archive-start":"00000001000001D7000000B1","backup-archive-stop":"00000001000001D7000000B1","backup-info-repo-size":52190861,"backup-info-repo-size-delta":11379901,"backup-info-size":146687162,"backup-info-size-delta":60255432,"backup-prior":"20150803-143249F","backup-reference":["20150803-143249F"],"backup-timestamp-start":1438743602,"backup-timestamp-stop":1438743607,"backup-type":"diff","db-id":1,"option-archive-check":true,"option-archive-copy":true,"option-compress":true,"option-hardlink":false,"option-start-stop":true}

[db]
db-catalog-version=201306121
db-control-version=937
db-id=1
db-system-id=6023966503359999454
db-version="9.3"

[db:history]
1={"db-catalog-version":201306121,"db-control-version":937,"db-system-id":6023966503359999454,"db-version":"9.3"}
backrest@prd-ba-01-tor ~ $
```

**Comment by dwsteele** [comment]
Fixed in 8b57188.

I don't think this bug is quite worth a release but one is coming pretty soon.  Let me know if this is a serious issue for you.  The json output is correct in the current version.
